spec/models/patient_spec.rb

```diff
@@ -199,6 +203,98 @@ def refresh_views
 
         expect(described_class.ltfu_as_of(Time.current)).not_to include(not_ltfu_patient)
       end
+
+      describe "timezone-specific boundaries" do
+        def with_reporting_time_zones(&blk)
+          Time.use_zone(reporting_timezone) do
+            Groupdate.time_zone = reporting_timezone
+            blk.call
+            Groupdate.time_zone = nil
+          end
+        end
+
+        it "bp cutoffs for a year ago" do
+          # For any provided date in June in the local timezone, the LTFU BP cutoff is the end of June 30 of the
+          # previous year in the local timezone.
+          #
+          # Eg. For any date provided in June 2021, the cutoff is the June 30-Jul 1 boundary of 2020
+
+          long_ago = 5.years.ago
+
+          # We explicitly set the times in the reporting TZ here, but don't use the block helper because its a hassle w/
+          # all the local vars we need
+          timezone = Time.find_zone(reporting_timezone)
```

Comment:
We get the zone object to use it explicitly, so we don't need to use the block helper here...the default time zone is UTC, which is what we want for the actual fixture creation below.